pkgs/data/themes/sddm-sugar-dark/default.nix

```
@@ -0,0 +1,47 @@
+{ pkgs, lib, stdenvNoCC, themeConfig ? null }:
+
+let
+  name = "sugar-dark";
+  owner = "MarianArlt";
+
+  basePath = "$out/share/sddm/themes/${name}";
+  iniFormat = pkgs.formats.ini { };
+in
+
+stdenvNoCC.mkDerivation rec {
+  pname = "sddm-${name}";
+  version = "1.2";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "${owner}";
+    repo = "${pname}";
+    rev = "v${version}";
```

```suggestion
stdenvNoCC.mkDerivation rec {
  pname = "sugar-dark";
  version = "1.2";

  src = pkgs.fetchFromGitHub {
    owner = "MarianArlt";
    repo = "sugar-dark";
    rev = "v${version}";
```
lets not go into templating hell

---

Sure I'll avoid the templates but we should probably use sddm-sugar-dark as a package name because

1. It allows users to search for sddm an still find the theme
2. It goes along with the AURs naming scheme. See [repology](https://repology.org/projects/?search=sugar-dark&maintainer=&category=&inrepo=&notinrepo=&repos=&families=&repos_newest=&families_newest=)